components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -61,7 +61,10 @@ export default function Preferences() {
 
             // TODO: Convert this to a mutation
             try {
-                await getGitpodService().server.updateWorkspaceTimeoutSetting({ workspaceTimeout: workspaceTimeout });
+                await getGitpodService().server.updateWorkspaceTimeoutSetting({
+                    workspaceTimeout: workspaceTimeout,
+                    disabledClosedTimeout: true,
+                });
```

Comment:
Need to check if it's default workspace timeout then enable close timeout again

Reply:
Handled it in [`f8d3f5e`](https://github.com/gitpod-io/gitpod/pull/19426/commits/f8d3f5e190a8accd50346a3a876ff5e3915edb1b)